index.js

```diff
@@ -12,6 +12,11 @@ export default {
     if (options.action && options.action.color) {
       options.action.color = processColor(options.action.color);
     }
+
+     if(!options.tintColor) {
```

Comment:
I'm not sure which components you're referring to, but I think generally tint is used for images (not text). Android also uses textColor to specify this on their [native snackbars](https://developer.android.com/reference/android/support/design/widget/Snackbar.html).